Server.Tests/BranchingStrategy/BranchTypeLookupShould.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Microsoft.Extensions.Options;
+
+namespace PrincipleStudios.ScaledGitApp.BranchingStrategy;
+
+public class BranchTypeLookupShould
+{
+	private readonly BranchTypeLookup branchTypeLookup;
+
+	public BranchTypeLookupShould()
+	{
+		var configuration = new BranchingStrategyConfiguration
+		{
+			DefaultBranchType = "feature",
+			BranchTypes = new Dictionary<string, BranchTypeConfiguration>
+			{
+				["main"] = new()
+				{
+					Patterns = ["^main$"],
+					Colors = ["rgb(112, 112, 112)"],
```

Comment:
Are these colors all really similar shades of grey?

Reply:
Yeah, just for tests, though.